keps/sig-release/1731-publishing-packages/README.md

```diff
@@ -236,11 +240,14 @@ Once the tests show that the mirrors are good, we can adapt the official documen
 
```

Comment:
I think it would be good to update alpha requirements to include:
- There is a documented process to create and publish deb and rpm packages of Kubernetes components
- It is possible to consume the published deb and rpm packages using steps similar to the documented process

Reply:
Yes to this! Why 'similar' though, should the docs not offer a definitive guide?

Reply:
> Yes to this! Why 'similar' though, should the docs not offer a definitive guide?

My thought was that for Alpha we would not update the docs to reference the new package repositories. In which case some modifications would be needed for testing to work around the docs referencing old gpg keys, repositories, etc.